README.md

```diff
@@ -26,6 +26,23 @@ If you want to run a development server, you can use:
 yarn watch
 ```
 
+## Contributing
+
+First of all, contributions are very welcome! Please open a pull request and be available, so we can ask follow-up questions on your contribution, and be aware of the [licence](#license) the project is published under.
```

Comment:
```suggestion
First of all, contributions are very welcome! Please open a pull request and be available, so we can ask follow-up questions on your contribution, and be aware of the [license](#license) the project is published under.
```

Why do you insist on this spelling that is inconsistent with this repo (as well as [all of our other repos](https://github.com/search?q=licence+%28user%3Abaltpeter+OR+user%3Azner0l+OR+org%3Adatenanfragen+OR+org%3AtweaselORG%29&type=code&p=1)) and barely used by anyone in open source dev (https://github.com/search?q=path%3ALICENSE&type=code vs. https://github.com/search?q=path%3ALICENCE&type=code)?